ui/apps/platform/src/Containers/Violations/ViolationsTableSearchFilter.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Toolbar, ToolbarGroup, ToolbarContent, ToolbarItem } from '@patternfly/react-core';
+
+import useURLSearch from 'hooks/useURLSearch';
+import {
+    makeFilterChipDescriptors,
+    onURLSearch,
```

Comment:
Hmm I'm not sure if I am using this helper for the main VM pages. If not, I'll see if I can get a chance to refactor so that we aren't duplicating logic.